configs/log_dir/default.yaml

```diff
@@ -2,7 +2,7 @@
 
 hydra:
   run:
-    dir: /ssd1/jiahe/logs/experiments/runs/${name}/${now:%Y-%m-%d}_${now:%H-%M-%S}
+    dir: /ssd1/jiahe/logs/experiments/runs/${name}/${now:%Y-%doc2json-%d}_${now:%H-%M-%S}
```

Comment:
why change to doc2json from m? By mistake?

Reply:
Sorry I did it by mistake, I just got lazy. I created a test directory at the first and named it as 'm', and when I renamed it the m here was changed...

Reply:
Previous code indicates a date by %Y-%m-%d. You should not change it.